--- Makefile.orig	2019-06-16 17:12:52.503192924 +0200
+++ Makefile	2019-06-16 17:13:08.626244896 +0200
@@ -2,8 +2,6 @@
 DEP	=	dep
 EXE = ${OBJ}/bin
 
-COMMIT := $(shell git log -1 --pretty=format:"%H")
-
 ARCH =
 ifeq ($m, 32)
 ARCH = -m32
@@ -12,7 +10,7 @@
 ARCH = -m64
 endif
 
-CFLAGS = $(ARCH) -O3 -std=gnu11 -Wall -mpclmul -march=core2 -mfpmath=sse -mssse3 -fno-strict-aliasing -fno-strict-overflow -fwrapv -DAES=1 -DCOMMIT=\"${COMMIT}\" -D_GNU_SOURCE=1 -D_FILE_OFFSET_BITS=64
+CFLAGS = $(ARCH) -O3 -std=gnu11 -Wall -mpclmul -march=core2 -mfpmath=sse -mssse3 -fno-strict-aliasing -fno-strict-overflow -fwrapv -DAES=1 -D_GNU_SOURCE=1 -D_FILE_OFFSET_BITS=64
 LDFLAGS = $(ARCH) -ggdb -rdynamic -lm -lrt -lcrypto -lz -lpthread -lcrypto
 
 LIB = ${OBJ}/lib
